test/epoch_token_locker.js

```diff
@@ -191,7 +191,28 @@ contract("EpochTokenLocker", async (accounts) => {
       assert.equal(await epochTokenLocker.getBalance(user_1, ERC20.address), 100)
     })
   })
-  describe("subtractBalance", () => {  
+  describe("addBalanceAndPostponeWithdraw", () => {
+    it("does not postpone a withdrawRequest for a future epoch", async () => {
+      const epochTokenLocker = await EpochTokenLockerTestInterface.new()
+      const ERC20 = await MockContract.new()
+      const currentStateIndex = await epochTokenLocker.getCurrentStateIndex.call()
+      await epochTokenLocker.requestWithdraw(ERC20.address, 100, { from: user_1 })
+      await epochTokenLocker.addBalanceAndPostponeWithdrawTest(user_1, ERC20.address, 100)
+
+      assert.equal((await epochTokenLocker.getPendingWithdrawBatchNumber(user_1, ERC20.address)).toNumber(), currentStateIndex, "state index was falsely updated")
```

Comment:
Although this will never happen, I wonder about slightly more description in the comment... Like 

"State index updated incorrectly during call to addBalanceAndPostponeWithdrawTest"